src/main/openapi/ds-image-analysis-openapi_v1.yaml

```diff
@@ -44,15 +39,70 @@ paths:
 
       responses:
         '200':
-          description: 'The dominant colors'
+          description: 'The dominant color'
           content:
-            application/json:
+            text/plain:
               schema:
                 type: string
-              example:
-                color-one: '#6b33ff'
-                color-two: '#6ab044'
-                color-three: '#a56c13'
+                example: "#000000"
+
+  /colorcount:
+    post:
+      tags: 
+        - 'Image Analysis'
+      summary: 'Count the unique colors in the given image.'
+      operationId: getColorCount
+      requestBody:
+        content:
+          multipart/form-data:
+              schema:
+                type: object
+                required:
+                  - image
+                properties:
+                  # The POSTed image can be retrieved using imageDetail.getDataHandler().getInputStream()
+                  image:
+                    description: 'The image to analyse'
+                    type: string
+                    format: binary
+      responses:
+          '200':
+            description: 'Number of unique colors in image:'
+            content:
+              text/plain:
+                schema:
+                  type: integer
```

Comment:
Remember to stats the `format` when using an integer (`int32` is fine for unique colors).

Reply:
Format has been added